examples/ts-avo-inspector-example/src/App.test.tsx

```diff
@@ -2,6 +2,30 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import App from './App';
 
+jest.mock('avo-inspector', () => {
+  const path = require('path');
+  const fs = require('fs');
+
+  const moduleName = 'avo-inspector';
+  const modulePath = require.resolve(moduleName);
+  const packageJsonPath = path.resolve(modulePath, '..', '..', 'package.json');
+
+  const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'));
+
+  if (packageJson.browser) {
+    const browserPath = path.resolve(path.dirname(packageJsonPath), packageJson.browser);
+    const nodeModulesIndex = browserPath.indexOf('node_modules');
+    const startIndex = nodeModulesIndex + 'node_modules/'.length;
+    const relativePath = browserPath.substring(startIndex);
+
+    const browserModule = jest.requireActual(browserPath);
+
+    return browserModule;//require("avo-inspector/dist/index.js");
```

Comment:
I that comment supposed to be there?
```suggestion
    return browserModule; // require("avo-inspector/dist/index.js");
```